docs/05-reference/api/delete-asset.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "DELETE asset"
+---
+
+# `DELETE` asset
+
+`DELETE /asset/{ASSET_ID}` deletes an existing `asset` object.
+
+This will delete an existing asset from the Livepeer API database. Deleted
+assets can currently be played back from a playback URL, even if the asset is
+deleted. There is currently no way to delete an asset from IPFS or "unpin" the
+asset through Pinata.
```

Comment:
- Should more clearly disclaim that the file is not deleted from storage, only the links to it in Studio DB.
 - Pinata should not be exposed to users, it's an implementation detail on how we save things on IPFS
 - IPFS/Pinata is as much deletable as any other centralized storage, no need for a special disclaim apart from how IPFS itself works, which we don't need to mention here. If we get a request to delete, we should delete from every storage we have.

I'd rewrite this as
```suggestion
This will delete an existing asset from the Livepeer API database. Keep in mind that this does not delete the contents of the asset from the storage. This means that deleted assets can still be accessed through their download and playback URLs.
```